From: SeongJae Park <sj@kernel.org>
Date: Sat, 29 Nov 2025 08:54:21 -0800
Subject: [PATCH] mm/damon: fix repeated word on report_filter kernel doc

This will be squashed into the commit that introduced the repeated word,
before being posted to the mailing list.

Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/aSrWuUuXhjnUZu2W@rli9-mobl
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 include/linux/damon.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/damon.h b/include/linux/damon.h
index a62f3ba980df..ba27349b3a74 100644
--- a/include/linux/damon.h
+++ b/include/linux/damon.h
@@ -859,7 +859,7 @@ enum damon_report_filter_type {
 /**
  * struct damon_report_filter - &struct damon_access_report filter.
  * @type:	The type of this filter.
- * @matching:	Whether is is for condition-matching reports.
+ * @matching:	Whether it is for condition-matching reports.
  * @allow:	Whether to include or excludie the @matching reports.
  * @cpus:	Access-originated CPUs if @type is DAMON_FILTER_TYPE_CPUS.
  * @list:	List head for siblings.
-- 
2.47.3

